packages/docusaurus-theme-classic/src/theme/DocSidebarItem/index.tsx

```diff
@@ -139,7 +139,13 @@ function DocSidebarItemCategory({
         },
         className,
       )}>
-      <div className="menu__list-item-collapsible">
+      <div
+        className={clsx(
+          ThemeClassNames.docs.docSidebarItemCategoryLink,
+          ThemeClassNames.docs.docSidebarItemCategoryLinkLevel(level),
```

Comment:
do we really need those extra classes stable? What's the purpose?

I know we want stable selectors but IMHO on parent it's good enough, it's quite easy to target the first child div once we can target the parent reliably